UIKit-Template/UIKit-Template/Models/PostTableViewCell.swift

```diff
@@ -0,0 +1,293 @@
+            likesLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 13),
+            likesLabel.widthAnchor.constraint(equalToConstant: 107),
+            likesLabel.heightAnchor.constraint(equalToConstant: 15),
+
+            postText.topAnchor.constraint(equalTo: likesLabel.bottomAnchor, constant: 6),
+            postText.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 13),
+            postText.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 10),
+            postText.heightAnchor.constraint(equalToConstant: 30),
+
+            myAvatar.widthAnchor.constraint(equalToConstant: 20),
+            myAvatar.heightAnchor.constraint(equalToConstant: 20),
+            myAvatar.topAnchor.constraint(equalTo: postText.bottomAnchor, constant: 6),
+            myAvatar.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 13),
+
+            commentTextField.centerYAnchor.constraint(equalTo: myAvatar.centerYAnchor),
+            commentTextField.leadingAnchor.constraint(equalTo: myAvatar.trailingAnchor, constant: 3),
+            commentTextField.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 10),
+            commentTextField.heightAnchor.constraint(equalToConstant: 15),
+
+            timeLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 13),
+            timeLabel.topAnchor.constraint(equalTo: myAvatar.bottomAnchor, constant: 4),
+            timeLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 10),
+            timeLabel.heightAnchor.constraint(equalToConstant: 15),
+            timeLabel.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -20)
+        ])
+    }
+
+    private func configurateScrollView() {
+        scrollView.showsHorizontalScrollIndicator = false
+
+        scrollView.backgroundColor = .white
+        scrollView.contentSize = CGSize(width: UIScreen.main.bounds.width * CGFloat(postImages.count), height: 239)
+        scrollView.isPagingEnabled = true
+        scrollView.delegate = self
+
+        for (index, image) in postImages.enumerated() {
+            addImage(image: image, position: CGFloat(index))
+        }
+
+        pageControl.numberOfPages = postImages.count
+        pageControl.pageIndicatorTintColor = .lightGray
+        pageControl.currentPageIndicatorTintColor = .black
+    }
+
+    private func addImage(image: UIImage, position: CGFloat) {
+        let imageView = UIImageView()
+        imageView.image = image
+        scrollView.addSubview(imageView)
+        let screenWidth = UIScreen.main.bounds.width
+        imageView.frame = CGRect(x: screenWidth * position, y: 0, width: screenWidth, height: 239)
+    }
+
+    @objc private func actionsButtonTapped() {}
+
+    @objc private func likeButtonTapped() {}
+
+    @objc private func commentsButtonTapped() {}
+
+    @objc private func shareButtonTapped() {}
+
+    @objc private func bookmarkButtonTapped() {}
+}
+
+// MARK: - Extension UIScrollViewDelegate
```

Comment:
```suggestion
// MARK: - MainTableViewCell + UIScrollViewDelegate
```
и проверь везде чтобы так были подписаны